test/helper.js

```diff
@@ -110,29 +122,34 @@ class Helper {
 
     args = [...args, '--ua', 'pear/terminal']
 
-    const di = args.findIndex(arg => arg.startsWith('--debug='))
-    if (di > -1) args.push(args.splice(di, 1)[0])
-
     const iterable = new Readable({ objectMode: true })
 
-    const child = spawn(this.runtime, args, {
-      stdio: silent ? 'ignore' : ['inherit', 'pipe', 'pipe']
+    this.app = spawn(this.runtime, args, {
```

Comment:
can you guarantee that the helper will only ever spawn one app at a time and `this.app` won't be overwritten by another spawn (thus losing the ref)?

Reply:
do `yield { tag: 'child', child }`